packages/db/src/models/granule.ts

```diff
@@ -34,18 +34,23 @@ export default class GranulePgModel extends BasePgModel<PostgresGranule, Postgre
           updated_at: granule.updated_at,
           created_at: granule.created_at,
         })
-        .where(knexOrTrx.raw(`${this.tableName}.created_at <= to_timestamp(${translateDateToUTC(granule.created_at)})`))
+        .where(knexOrTrx.raw(`${this.tableName}.created_at <= to_timestamp(${translateDateToUTC(granule.created_at)})`));
+
+      if (executionCumulusId) {
```

Comment:
The only place where `executionCumulusId` could be undefined is from the data migrations

Reply:
d26488cb7be5f7a1d13c2910275e3aaef1acf4a7